lib/helpers.js

```diff
@@ -57,12 +90,28 @@ function assigneeFilter (pr, username) {
 
 /**
  * Build a message to send back to slack
+ * @param  {Array<BranchAndComparison>} branches List of all branches and their
+ *  comparison to the repo's master branch
  * @param  {Array<PR>} prs List of all PRs
- * @param  {SlackMessage} [username] If included, the PRs will be
- * filtered according to which PRs are assigned to a particular user
+ * @param  {String} [username] If included, the PRs will be filtered according
+ *  to which PRs are assigned to a particular user
  * @return {Promise} Resolve type can be either a string or array of strings
  */
-function buildMessage (prs, username) {
+function buildMessage (branches, prs, username) {
```

Comment:
I have to pause for a minute each time I see `prs` and tell myself it means `PRs`, not `P.R.S.`... I don't know if `prList` might be a better name.

Reply:
Or `pulls`